a1aa288

commit a1aa288
Author: Athanatos123 <alain.delree@gmail.com>
Date:   Tue Aug 25 10:13:31 2026 +0200

    [code] Nettoie .gitignore : retire FF_Galerie_projet.md et Notes perso/ (docs perso déménagés) (#103)

diff --git a/.gitignore b/.gitignore
index bf52c73..32fdc04 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,10 +1,8 @@
 data/prix.json
 node_modules/
 .DS_Store
-FF_Galerie_projet.md
 
 # Notes et fichiers de travail personnels
-Notes perso/
 Site Fred.md
 *.old.*
 node_modules
